### DISCUSSION TITLE
Build Controls

### DISCUSSION THREAD

**Discussion by Influx-Davin** [lurodsil/ring-engine#4] [Q&A]
Does anyone know how to fix the build Controls please.
Game controls don't seem to work after build.
Any help anyone ?

**Comment by lurodsil** [comment]
In the project settings folder have a xml InputManagerRuntime.xml, you have to copy to project data folder after build.

**Comment by Influx-Davin** [comment]
Thanks, I'll try it. And sorry for not replying.

On Saturday, 24 July 2021, Lucas Rodrigues ***@***.***> wrote:
> In the project settings folder have a xml InputManagerRuntime.xml, you
have to copy to project data folder after build.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or unsubscribe.<
https://ci6.googleusercontent.com/proxy/VFyulxS-GXQ0hssvcr5mkw4HzUqn3bW8RGTcXk0BmhTcGJ2nawvNbWavFSPm29SoUTS_BrON5rXQDwTiSp_db_6vsUB-VYgrjZYZpw5vnsIROef7mYW7bZ6txCoFuappvkAJilqLizXL0WnJUrXFB1SBn31nUAQwbY_4DzrTiujtaXEswmJydQu-vxyQKO58L9dEqPzXIO3loo9W0C_WieyKLMyPjOwU2yMzZzC7IsCIBEwvfDNy=s0-d-e1-ft#https://github.com/notifications/beacon/AUWS3YVIXH3V7UVY4CTSZTDTZK627A5CNFSM5APHLZOKYY3PNVWWK3TUL52HS4DFWFCGS43DOVZXG2LPNZBW63LNMVXHJKTDN5WW2ZLOORPWSZGOAAH7GXY.gif>
